server/jobs/index.js

```diff
@@ -1,22 +1,73 @@
 /* eslint-disable import/no-dynamic-require */
 /* eslint-disable global-require */
+/* eslint-disable  no-restricted-syntax */
+/* eslint-disable no-await-in-loop */
+
 import fs from 'fs';
+import ms from 'ms';
+
+import { placementStatus, fetchNewPlacements } from '../modules/allocations';
+import client from '../helpers/redis';
 
 /**
  * @function getJobs
  * @param {string} folderName The folder containing the jobs.
  * @returns {array} An array of function representing jobs.
  */
 const getJobs = (folderName) => {
-  const jobs = fs
-    .readdirSync(`${__dirname}/${folderName}`)
-    .map((fileName) => {
-      const { default: exported } = require(`${__dirname}/${folderName}/${fileName}`);
-      return typeof (exported) === 'function' ? exported : () => {};
-    });
+  const jobs = fs.readdirSync(`${__dirname}/${folderName}`).map((fileName) => {
+    const { default: exported } = require(`${__dirname}/${folderName}/${fileName}`);
+    return typeof exported === 'function' ? exported : () => {};
+  });
   return jobs;
 };
 
-export const offboardingJobs = getJobs('offboarding');
+const jobs = {
+  offboarding: {
+    jobList: getJobs('offboarding'),
+    allocationStatus: 'rollingOff',
+    automationResult: {
+      type: 'Offboarding',
+    },
+  },
+  onboarding: {
+    jobList: getJobs('onboarding'),
+    allocationStatus: 'onboarding',
+    automationResult: {
+      type: 'Onboarding',
+    },
+  },
+};
 
-export const onboardingJobs = getJobs('onboarding');
+/**
+ * @desc Executes jobs to automate developer offboarding/onboarding tasks
+ *
+ * @param {string} type Type of job to execute: offboarding || onboarding
+ *
+ * @returns {void}
+ */
+export default function executeJobs(type) {
+  const { jobList, allocationStatus, automationResult } = jobs[type];
+  const status = placementStatus[allocationStatus];
+  let fetchPlacementError;
+  return fetchNewPlacements(status, 3)
```

Comment:
Why are we using `3` here, is it not supposed to be `1`??

Reply:
Was used during testing, will revert